### PR TITLE
PointPillars inference pipeline

### PR DESCRIPTION
**Pull request by Prantl**
This PR contains:
- Implementation of PointPillars (MMDetection3D as reference)
- Inference pipeline with small test script (with visualization)
- Implementation of three loss functions (FocalLoss, Smooth_L1, weighted CrossEntropy)

### CODE REVIEW

File: ml3d/torch/models/point_pillars.py [hunk 1]
```diff
@@ -0,0 +1,795 @@
...
+MODEL._register_module(PointPillars, 'torch')
+
+
+class PointPillarsVoxelization(torch.nn.Module):
+
+    def __init__(self,
+                 voxel_size,
+                 point_cloud_range,
+                 max_num_points=32,
+                 max_voxels=[16000, 40000]):
```
Comment (sanskar107) [inline ml3d/torch/models/point_pillars.py:210]: We should change the default argument to `tuple`, as we check `isinstance(max_voxels, tuple)` in line `228`

**Review by sanskar107 — APPROVED** [review]
